src/earn/EarnEnterAmount.tsx

```diff
@@ -563,34 +457,34 @@ function EarnEnterAmount({ route }: Props) {
           swapTransaction={swapTransaction}
           pool={pool}
           token={inputToken}
-          tokenAmount={tokenAmount}
+          tokenAmount={processedAmounts.token.bignum}
           isWithdrawal={isWithdrawal}
         />
       )}
-      {swapTransaction && tokenAmount && (
+      {swapTransaction && processedAmounts.token.bignum && (
         <SwapDetailsBottomSheet
           forwardedRef={swapDetailsBottomSheetRef}
           testID="SwapDetailsBottomSheet"
           swapTransaction={swapTransaction}
           token={inputToken}
           pool={pool}
-          tokenAmount={tokenAmount}
-          parsedTokenAmount={parsedTokenAmount}
+          tokenAmount={processedAmounts.token.bignum}
+          parsedTokenAmount={processedAmounts.token.bignum}
```

Comment:
I'm not sure if there's any real need for these being different props but I've kept it as it was to not change implementation of the subsequent components.